packages/optimizer/README.md

```diff
@@ -226,7 +209,15 @@ Minifies the generated HTML output and inlined CSS.
 
 #### `preloadHeroImage`
 
-Auto detect hero images for amp-img, amp-iframe, amp-video, or amp-video-iframe and injects a `link rel=preload`. Preloads will only be generated if there is no existing image preload.
+Enables hero image optimization. Hero images will either be auto detected or you can explicitly mark these by adding the `data-hero` attribute:
+
+```
+<amp-img data-hero src="foo.jpg" ...>
+```
+
+The maximum number of hero images that can be marked up using `data-hero` is `2`.
+
+If no `data-hero` attribute is present, AMP optimizer auto detects hero images for `amp-img`, `amp-iframe`, `amp-video`, or `amp-video-iframe` and injects a `link rel=preload`. Preloads will only be generated if there is no existing image preload. For `amp-img` elements, it will also server-side render the `img` element inside the `amp-img` element. This greatly improves image rendering performance.
```

Comment:
```suggestion
If no `data-hero` attribute is present, AMP optimizer auto-detects hero images for `amp-img`, `amp-iframe`, `amp-video`, or `amp-video-iframe` and injects a `link rel=preload`. Image preload links will only be generated if there is none already existing. For `amp-img` elements, it will also server-side render the `img` element inside the `amp-img` element. This greatly improves image rendering performance and reduces the [largest contentful paint](https://web.dev/lcp/) (LCP) metric from [Core Web Vitals](https://web.dev/vitals/).
```